### ISSUE TITLE
Sword of the Zephyr damage may be weird

### ISSUE THREAD

**Issue by Dayyer1** [Azanor/thaumcraft-beta#302]
I use a mod that levels up items depending on how long you use them.

[https://minecraft.curseforge.com/projects/levels?gameCategorySlug=mc-mods&projectID=235552](url)

Part of the mod makes it so that, the higher level a weapon becomes, the more damage it does.

So, currently, my Sword of the Zephyr does 52.61 damage, but when a hit a mob, it only applies ~14 damage, even if it is a single mob and the arcing effect is not applied.

I also have a diamond axe that does 18.96 damage, with Arcing 1. The thing is, the axe consistently does 18.96 damage.

TL;DR: Sword of the Zephyr damage is weird.

@Azanor I am sorry for putting so many issues in your git ;-;
Thanks for your work, and good luck with fixing everything. <3
